code/game/objects/items/weapons/grenades/frag.dm

```diff
@@ -90,4 +125,9 @@
 	qdel(src)
 	return TRUE
 
+/obj/item/shrapnel/holy
+	name = "blessed shrapnel"
+	desc = "Shards of godlike judgement, launched at high velocity. The power of Christ propels you."
+	force = 6
```

Comment:
It would be interesting if these had a unique visual, even if its just `color = "#FFFF00"`